odoo/addons/base/models/ir_http.py

```diff
@@ -286,6 +286,10 @@ def _clear_routing_map(cls):
     # Binary server
     #------------------------------------------------------
 
+    def _get_models_whose_record_name_likely_filename(self):
+        # List of models whose name is likely to be a file name with extension.
+        return {'ir.attachment'}
```

Comment:
That doesn't seem to make much sense. At that point all models will have to be excluded, especially since the OPW discusses custom fields.